src/main/kotlin/com/example/Employee/controllers/HolidaysController.kt

```diff
@@ -19,23 +18,23 @@ import java.util.*
 public class HolidaysController (private var holidaysService: HolidaysService){
 
     @PostMapping
-    fun createHoliday(@RequestBody holiday: Holidays):ResponseEntity<Holidays> {
+    fun createHoliday(@RequestBody holiday: Holiday):ResponseEntity<Holiday> {
         holidaysService.createHoliday(holiday)
         return ResponseEntity.ok(holiday)
     }
 
     @GetMapping
-    fun getHolidays(): MutableIterable<Holidays> {
+    fun getHolidays(): MutableIterable<Holiday> {
         return holidaysService.getHolidays()
     }
 
     @GetMapping("/{id}")
-    fun getHoliday(@PathVariable id: Int): Optional<Holidays> {
+    fun getHoliday(@PathVariable id: Int): Optional<Holiday> {
         return holidaysService.getHoliday(id)
     }
 
     @PutMapping("/{id}")
-    fun updateHoliday(@PathVariable id: Int, @RequestBody holiday: Holidays): ResponseEntity<Holidays> {
+    fun updateHoliday(@PathVariable id: Int, @RequestBody holiday: Holiday): ResponseEntity<Holiday> {
```

Comment:
You've used ResponseEntity here, but in other places you've not. Its best to use ResponseEntity for everything. `ResponseEntity` allows a lot of control over the response.